app/tray/Account/Account/Requests/SignTypedDataRequest/Default/index.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+
+const SimpleJSON = ({ json }) => {
```

Comment:
can we just keep this code in the index of the parent folder? not sure we need something called "default" as that's implied by the index.js